torch_xla/core/xla_env_vars.py

```diff
@@ -1,17 +1,15 @@
 MP_DEVICE = 'XRT_MULTI_PROCESSING_DEVICE'
-LOCAL_WORKER = 'XRT_LOCAL_WORKER'
 TPU_CONFIG = 'XRT_TPU_CONFIG'
 TPUVM_MODE = 'TPUVM_MODE'
 SERVICE_ADDRESS = 'XRT_MESH_SERVICE_ADDRESS'
 DEVICE_MAP = 'XRT_DEVICE_MAP'
-WORKERS = 'XRT_WORKERS'
-LOCAL_ORDINAL = 'XRT_SHARD_LOCAL_ORDINAL'
-ORDINAL = 'XRT_SHARD_ORDINAL'
+WORKERS = 'RT_WORKERS'
+LOCAL_ORDINAL = 'RT_SHARD_LOCAL_ORDINAL'
+ORDINAL = 'RT_SHARD_ORDINAL'
```

Comment:
do we still need these? A lot of them are used by XRT only. can you find out who set these env var?

Reply:
these var are added 4 years ago:
https://github.com/pytorch/xla/blame/b3f4c7bc1e9f812069770c83e38e5bd2dc20aef6/torch_xla/core/xla_env_vars.py#LL7-L9C30

could create a ticket to track whether we still need these env var or not

Reply:
has rebased and these veriables has been deleted before rebase